core/common/package.json

```diff
@@ -38,7 +38,8 @@
     "@ungap/url-search-params": "^0.1.2",
     "flatbuffers": "~1.12.0",
     "semver": "^5.5.0",
-    "js-base64": "^3.6.1"
+    "js-base64": "^3.6.1",
+    "is-buffer": "^2.0.5"
```

Comment:
I don't think we should add a new dependency on this package. It looks to me like a one-liner. Just copy it and give credit in `ThirdPartyNotices.md`

Reply:
fair enough

Reply:
@kabentley I added the function